src/xrEngine/IGame_Persistent.cpp

```diff
@@ -25,7 +25,7 @@ bool IGame_Persistent::IsMainMenuActive()
 
 bool IGame_Persistent::MainMenuActiveOrLevelNotExist()
 {
-    return !g_pGameLevel || g_pGamePersistent->m_pMainMenu && g_pGamePersistent->m_pMainMenu->IsActive();
+    return !g_pGameLevel || (g_pGamePersistent->m_pMainMenu && g_pGamePersistent->m_pMainMenu->IsActive());
```

Comment:
Did we agreed on disabling that warning or leaving it enabled?...

Reply:
Yes, this one just slipped into this PR :) removed